feedback/static/js/src/feedback.js

```diff
@@ -42,6 +42,13 @@ function FeedbackXBlock(runtime, element) {
 		$('.feedback_thank_you', element).text(data.response);
 	    }
         });
+
+	if(vote >= 9) {
+		alert(
+			$('.feedback_likert_scale', element).data("external-feedback") +
```

Comment:
`$('.feedback_likert_scale', element)` is used twice here.
Would it render correctly? Will the message be like `Test messagehttps://example.com`? Do we need to add blank space between two elements?

Reply:
This is draft 'alert' for task work checking. 
I will transfer this ticket to Peter Kulko and he will make custom `pop-up` instead this `alert` and will replace these lines.